app/views/companies/_navbar.html.erb

```diff
@@ -0,0 +1,41 @@
+<nav class="navbar navbar-expand-md navbar-dark fixed-top bg-dark">
+   <div class="container-fluid">
+
+      <a class="navbar-brand" href="<%= root_path %>">JSCLook</a>
+
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse"
+         aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
+         <span class="navbar-toggler-icon"></span>
+      </button>
+
+      <div class="collapse navbar-collapse" id="navbarCollapse">
+         <ul class="navbar-nav me-auto mb-2 mb-md-0">
+            <li class="nav-item active">
+               <a class="nav-link" href="<%= root_path %>">Home </a>
+            </li>
+            <li class="nav-item">
+               <a class="nav-link" href="https://www.gpw.pl/">GPW</a>
+            </li>
+            <li class="nav-item">
+               <a class="nav-link" href="https://newconnect.pl/">NewConnect</a>
+            </li>
+         </ul>
+
+         <!--
+         <div  class="mx-auto">
+            <form class="d-flex">
+               <input class="form-control mr-sm-2" type="text" placeholder="Search" aria-label="Search">
+               <button class="btn btn-outline-secondary btn-sm" type="submit" role="button">Search</button>
+            </form>
+         </div> -->
+
```

Comment:
Do you need it?

```suggestion
```

Reply:
No, this is one of future possibility, but not needed now, will delete it